### PR TITLE
Update MarketDataMessage.capnp

### PR DESCRIPTION
**Pull request by calvinleungyk**

### CODE REVIEW

File: MarketDataMessage.capnp [hunk 1]
```diff
@@ -52,13 +52,13 @@ struct MarketDataRequest { # http://www.fixwiki.org/fixwiki/MarketDataRequest/FI
 #######################################################################################################
 
 struct MarketDataSnapshot { # http://fixwiki.org/fixwiki/MarketDataSnapshotFullRefresh/FIX.5.0SP2%2Bol
-    timestamp @0 :Float64;                  # required
```
Comment (vitaliybezz) [inline MarketDataMessage.capnp]: [Inquiry]
Is this message still in use? I am planning to refactor it from a scratch. If someone is using it, I need to create another file.

Comment (calvinleungyk) [reply MarketDataMessage.capnp]: @vitaliybezz Which message? Snapshot?

Comment (vitaliybezz) [reply MarketDataMessage.capnp]: I mean all MarketDataMessage.capnp

Comment (calvinleungyk) [reply MarketDataMessage.capnp]: Yes this is being used right now

Comment (vitaliybezz) [reply MarketDataMessage.capnp]: ok

**Pull request merged**